Modules/Video/Core/wrapping/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+itk_wrap_module(ITKVideo)
+
+set(WRAPPER_SUBMODULE_ORDER
```

Comment:
This is only needed if order of compilation is important (something needs to go before something else).

Reply:
Understood - order of compilation is important now that we're adding parent class wrappings as well.